content/productivity/distribute-silver/plugin.js

```diff
@@ -35,7 +36,31 @@ class Plugin {
       }
     }
 
-    let message = document.createElement('div');
```

Comment:
Crap... I shouldn't have deleted this line.

Reply:
Yeah, that needs to stay :D